test/WebJobs.Script.Tests.Integration/Storage/StorageEndToEndTests.cs

```diff
@@ -116,6 +116,26 @@ public async Task CanUseStorageV11Types()
             }, userMessageCallback: _fixture.Host.GetLog);
         }
 
+        [Fact]
+        public async Task CanUseStorageV12Types()
```

Comment:
@fabiocav do we want this behavior? I seem to recall that we discussed this a while ago and we didn't want to introduce any net-new storage types that can be used from csx without installing a storage extension... that we just added the V9 and V11 tests because our hands were tied. But I might be missing some key details.

Reply:
You're right. We'll have to make sure csx can't use these types. I'll keep the test around but make sure these v12 types aren't usable.